src/large_pages/node_large_page.cc

```diff
@@ -373,13 +371,16 @@ MoveTextRegionToLargePages(const text_region& r) {
     return -1;
   }
 #elif defined(__APPLE__)
+  // There is not enough room to reserve the mapping close
+  // to the region address so we content to give a hint
+  // without forcing the new address being closed to.
+  // We explicitally gives all permission since we plan
```

Comment:
Plan what? :-)

Can you also update the function comment? It currently says it always uses `MAP_FIXED`.

Reply:
Oups I forgot to save the rest of the comment.